child-logger.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var defaultLevels = require('./default-levels.js');
+var makeLogMethod = require('./log-method');
+var errors = require('./errors');
+
+module.exports = ChildLogger;
+
+function ChildLogger(config) {
+    this.mainLogger = config.mainLogger;
+    this.path = config.path;
```

Comment:
because the application developer calls `logger.childLogger('foo')` he would have to call it twice with the same 'foo'.

This should be an exception because it messes with our ability to change config for loggers based on paths on the fly.

However once we throw an exception the application developer can quickly resolve the namespace conflict because it should be obvouis.